.env.example

```diff
@@ -0,0 +1,7 @@
+SECRET_KEY = "YOUR APP SECRET KEY GOES HERE"
+
+# True - for development, False - for production
+DEBUG = True / False
```

Comment:
I'm not sure if `True / False` will work correctly, should we leave only one value?